.env.example

```diff
@@ -20,3 +20,5 @@ NEXT_PUBLIC_APP_URL=https://dapple-cms.docker:3000
 UNLILOGIN_PUBHUB_CLIENT_ID=XXX
 UNLILOGIN_PUBHUB_RETAILER_ID=XXX
 UNLILOGIN_PUBHUB_RETAILER_KEY_CODE=XXX
+UNLILOGIN_SERVICES_WS_USER=XXX
```

Comment:
Could we pls add all these relevant things into our onePass? Maybe with the same naming so that we all know where to find them even if we can't reach you.

Reply:
Good point!

Reply:
Made a "Go env.local" entry